src/pushNotifications/WasteReminder.ts

```diff
@@ -0,0 +1,126 @@
+import * as SecureStore from 'expo-secure-store';
+
+import { device, secrets, staticRestSuffix } from '../config';
+import * as appJson from '../../app.json';
+import { getPushTokenFromStorage, PushNotificationStorageKeys } from './TokenHandling';
+
+const namespace = appJson.expo.slug as keyof typeof secrets;
+
+type SettingInfo = {
+  city: string;
+  reminderTime: string;
+  onDayBefore: boolean;
+  street: string;
+  wasteType: string;
+  zip: string;
+};
+
+export const updateReminderSettings = async ({
+  city,
+  reminderTime,
+  onDayBefore,
+  street,
+  wasteType,
+  zip
+}: SettingInfo) => {
+  const accessToken = await SecureStore.getItemAsync('ACCESS_TOKEN');
+  const pushToken = await getPushTokenFromStorage();
+  const requestPath = secrets[namespace].serverUrl + staticRestSuffix.wasteReminderRegister;
+  const os =
+    device.platform === 'ios' || device.platform === 'android' ? device.platform : 'undefined';
```

Comment:
shouldn't `device.platform` be enough, as it returns the right one or something nullish otherwise?

Reply:
this way we pass `'undefined'` as a value to the backend in case something unhandled platform is used.
![Screenshot 2021-03-11 at 10 50 08](https://user-images.githubusercontent.com/59824597/110769479-811fa600-8258-11eb-9640-52e5afaed1e7.png)
![Screenshot 2021-03-11 at 10 50 20](https://user-images.githubusercontent.com/59824597/110769481-81b83c80-8258-11eb-8ef0-567e8fb2133f.png)